packs/spell-effects/spell-effect-dragon-form.json

```
@@ -0,0 +1,559 @@
+                        "predicate": [
+                            {
+                                "or": [
+                                    "dragon-form:conspirator",
+                                    "dragon-form:horned"
+                                ]
+                            }
+                        ],
+                        "value": "fortitude"
+                    },
+                    {
+                        "predicate": [
+                            {
+                                "or": [
+                                    "dragon-form:adamantine",
+                                    "dragon-form:diabolic",
+                                    "dragon-form:empyreal",
+                                    "dragon-form:fortune"
+                                ]
+                            }
+                        ],
+                        "value": "reflex"
+                    },
+                    {
+                        "predicate": [
+                            {
+                                "or": [
+                                    "dragon-form:mirage",
+                                    "dragon-form:omen"
+                                ]
+                            }
+                        ],
+                        "value": "will"
+                    }
+                ],
+                "key": "ChoiceSet",
+                "rollOption": "dragonbreath-save"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.dragonForm.damageDice",
+                "value": "(2*@item.level)-2"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.dragonForm.breathDamage",
+                "value": "{item|flags.pf2e.rulesSelections.dragonDamage}"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.dragonForm.dragonTradition",
+                "value": "{item|flags.pf2e.rulesSelections.dragonTradition}"
+            },
+            {
+                "key": "BattleForm",
+                "overrides": {
+                    "hasHands": true,
+                    "resistances": [
+                        {
+                            "type": "{item|flags.pf2e.rulesSelections.dragonDamage}",
+                            "value": 10
+                        }
+                    ],
+                    "senses": {
+                        "darkvision": {},
+                        "scent": {
+                            "acuity": "imprecise",
+                            "range": 60
+                        }
+                    },
+                    "speeds": {
+                        "fly": 100,
+                        "land": 40,
+                        "swim": 60
+                    },
```

```suggestion
                    },
                    "speeds": {
                        "fly": 100,
                        "land": 40
                    },
```
Swim speed snuck in?